source/infrastructure/router.coffee

```diff
@@ -0,0 +1,37 @@
+class Space.eventSourcing.Router extends Space.messaging.Controller
+
+  @ERRORS: {
+    aggregateNotSpecified: 'Please specify a Router::Aggregate class to be
+    managed by the router.'
+    missingCreateCommand: 'Please specify Router::CreateWith (a command class)
+    that will be used to create new instanes of the managed aggregate.'
+  }
+
+  Dependencies: {
+    repository: 'Space.eventSourcing.Repository'
+  }
+
+  Aggregate: null
+  CreateWith: null
```

Comment:
Ok got it :-) Either `creationCommand` or `initializingCommand`. It's good to keep away from 'create' when not in the imperative sense